ResearchKit/Common/ORKTaskViewController.m

```diff
@@ -910,10 +912,14 @@ - (void)showViewController:(ORKStepViewController *)viewController goForward:(BO
     
     NSString *progressLabel = nil;
     if ([self shouldDisplayProgressLabel]) {
-        ORKTaskProgress progress = [_task progressOfCurrentStep:viewController.step withResult:[self result]];
-        
-        if (progress.total > 0) {
-            progressLabel = [NSString stringWithFormat:ORKLocalizedString(@"STEP_PROGRESS_FORMAT", nil) ,ORKLocalizedStringFromNumber(@(progress.current + 1)), ORKLocalizedStringFromNumber(@(progress.total))];
+        if (viewController.parentReviewStep && viewController.parentReviewStep.isStandalone) {
+            progressLabel = viewController.parentReviewStep.title;
```

Comment:
Creating a `boolean hideProgressLabel` on ORKStepViewController to control its title bar behavior?